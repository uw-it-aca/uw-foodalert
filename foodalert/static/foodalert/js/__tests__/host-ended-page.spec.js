"use strict";
import Vue from 'vue';
import BootstrapVue from 'bootstrap-vue'
Vue.use(BootstrapVue);

import {mount} from '@vue/test-utils';
import EndedPage from '../pages/host/ended-page.vue';

describe('Host Ended Page', function () {
        test('renders correctly', function () {
            var wrapper = mount(EndedPage, {
            });
            expect(wrapper.element).toMatchSnapshot();
        });
})