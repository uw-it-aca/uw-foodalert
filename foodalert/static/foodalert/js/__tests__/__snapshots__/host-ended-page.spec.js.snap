// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Host Ended Page renders correctly 1`] = `
<div
  class="page"
>
  <div
    id="standard-notification"
  >
    <div
      class="collapse"
      id="notif-container"
      style="display: none;"
    >
      <div
        class="text-center py-3 container-fluid"
      >
        <div
          class="container"
        >
          <div
            class="row justify-content-center"
          >
            <div
              class="col-md-8"
            >
              <div
                class="h5 text-white pt-2"
              >
                
         Your update was sent.
    
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
   
  <div
    class="page-content"
  >
     
    <div
      class="container"
    >
      <div
        class="row justify-content-center"
      >
        <div
          class="page-content-padding col-md-8"
        >
          <h1
            class="pt-4 pb-2"
            id="standard-heading"
          >
            
        Our sincere thanks
    
          </h1>
           
          <div
            id="standard-body"
          >
            <p>
              Thank you for sharing food. Your generosity means that less people have to worry about food insecurity at UW. You've also contributed to decrease the University's food waste.
            </p>
          </div>
           
          <div
            class="mt-5"
          >
            <div
              class="row justify-content-between"
            >
              <div
                class="col-md-5 col-lg-4 order-md-2"
              >
                <button
                  class="btn mb-3 btn-primary btn-lg btn-block"
                  style="white-space: nowrap;"
                  type="submit"
                >
                  New notification
                </button>
              </div>
               
              <div
                class="col-md-5 col-lg-4 order-md-1"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
